src/views/mfa-verify/PushForm.js

```diff
@@ -58,12 +59,13 @@ define(['okta', 'util/CookieUtil'], function (Okta, CookieUtil) {
         deviceName: this.model.get('deviceName')
       });
     },
-    setSubmitState: function (ableToSubmit) {
+    setSubmitState: function (ableToSubmit, useRetryLabel) {
```

Comment:
From what I can tell, trans.poll is only called once in Factor.js. After this authjs sdk takes care of polling. So from that particular code in Factor.js I don't think we know how many PUSH_INTERVALs have passed.

Also, part of the ticket is to change the button text when the client side limit is hit, but yes, this differs from when the server side limit is hit.